guidelines/index.html

```diff
@@ -913,9 +913,9 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
      <li>テキストは、支援技術なしで 200％までサイズ変更でき、利用者が<a href="#dfn-on-a-full-screen-window" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-on-a-full-screen-window-1" title="最も普及したサイズのデスクトップやラップトップのディスプレイで、ビューポートを最大化した状態。">全画面表示</a>にしたウィンドウで 1 行のテキストを読むときに横スクロールする必要がない。</li>   
   </ul>
 
-<div class="note" role="note" id="issue-container-generatedID-5"><div role="heading" class="note-title marker" id="h-note-5" aria-level="5"><span>注記 1</span></div><p class="">Content is not required to use these values. The requirement is that a mechanism is available for users to change these presentation aspects. The mechanism can be provided by the browser or other user agent. Content is not required to provide the mechanism.</p></div>
+<div class="note" role="note" id="issue-container-generatedID-5"><div role="heading" class="note-title marker" id="h-note-5" aria-level="5"><span>注記 1</span></div><p class="">コンテンツがこれらの値を用いる必要はない。要求事項は、利用者がこれらの提示の外観を変更するメカニズムを利用できることである。このメカニズムは、ブラウザ又は他のユーザエージェントによって提供されることがある。コンテンツがメカニズムを提供する必要はない。</p></div>
```

Comment:
最後の部分、「必要はない」はやや強いかもしれないと思いました。ここでは「この達成基準の要求事項ではない」という意味ですが、「必要はない」と言ってしまうと「そんな機能はない方がいい」というようなニュアンスにも感じられそうで、すこし語弊がありそうです。
といいつつ、最初の文の "is not required" も同じか……。なんかまあいいかという気がしてきました。

それはそれとして、3文目では "The mechanism" を「このメカニズム」としているので、それに倣うと4文目の "the mechanism" も「このメカニズム」でしょうか。

```suggestion
<div class="note" role="note" id="issue-container-generatedID-5"><div role="heading" class="note-title marker" id="h-note-5" aria-level="5"><span>注記 1</span></div><p class="">コンテンツがこれらの値を用いる必要はない。要求事項は、利用者がこれらの提示の外観を変更するメカニズムを利用できることである。このメカニズムは、ブラウザ又は他のユーザエージェントによって提供されることがある。コンテンツがこのメカニズムを提供する必要はない。</p></div>
```

Reply:
「必要はない」の過剰な強さは自分も感じます。
「必ずしも要さない」だからそんな強くないじゃん、と思うのですが、「必要ない」だと「全然いらない」感じがあります。「必要“は”ない」だと「必ずしも」のニュアンスを多少帯びてくるのかな、と…。